SIS_import/sis_upload.rb

```diff
@@ -476,11 +513,20 @@ def create_instructor_sandbox_site(zip_file_name, outputFile)
 
 if (upload_error)
 	## check first about the environment variable setting for MAILTO '
-	p "Use the environment variable 'MAILTO' for sending out error messages to #{ENV['MAILTO']}"
+	p "Use the environment variable 'MAILTO' for sending out SIS upload error messages to #{ENV['MAILTO']}"
 	p upload_error
 	## send email to support team with the error message
-	`echo #{upload_error} | mail -s "#{$server} Upload Error" #{ENV['MAILTO']}`
+	`echo #{upload_error} | mail -s "#{$server} SIS Upload Error" #{ENV['MAILTO']}`
 else
+	if ($upload_warnings != "")
+		# mail the upload warning message
+		## check first about the environment variable setting for MAILTO '
+		p "Use the environment variable 'MAILTO' for sending out SIS upload warning messages to #{ENV['MAILTO']}"
+		p $upload_warnings
+		## send email to support team with the error message
+		`echo #{$upload_warnings}  | mail -s "#{$server} SIS Upload Warnings" #{ENV['MAILTO']}`
```

Comment:
The upload warning message also gets logged into log files. 

What "status code" do you refer here? Status code for email or status code for SIS upload?